Source/WebKit/Shared/WebCoreArgumentCoders.serialization.in

```diff
@@ -3923,3 +3937,8 @@ header: <WebCore/SecurityPolicyViolationEvent.h>
     unsigned long lineNumber();
     unsigned long columnNumber();
 };
+
```

Comment:
Is this needed?

Reply:
It's used by `WebCore::AuthenticationExtensionsClientOutputs::LargeBlobOutputs` above in this file